src/daemon/gateway.go

```diff
@@ -18,15 +18,15 @@ import (
 
 // GatewayConfig configuration set of gateway.
 type GatewayConfig struct {
-	BufferSize       int
-	DisableWalletAPI bool
+	BufferSize      int
+	EnableWalletAPI bool
 }
 
 // NewGatewayConfig create and init an GatewayConfig
 func NewGatewayConfig() GatewayConfig {
 	return GatewayConfig{
-		BufferSize:       32,
-		DisableWalletAPI: false,
+		BufferSize:      32,
+		EnableWalletAPI: false,
```

Comment:
EnableWalletAPI: true?

Reply:
It's disabled by default now, we enable it through run.sh and electron-main.js
The default settings should be the normal settings for running a server node